backend/tests/unit/graphql/mutations/test_webhook.py

```diff
@@ -68,6 +68,37 @@ async def test_create_webhook_with_node_kind_and_valid_node_event(
     assert webhook.event_type.value.value == "infrahub.node.created"
 
 
+async def test_update_webhook_with_optional_node_kind(
+    db: InfrahubDatabase, register_core_models_schema: None, default_branch: Branch
+) -> None:
+    gql_params = await prepare_graphql_params(db=db, include_subscription=False, branch=default_branch)
+    result = await graphql(
+        schema=gql_params.schema,
+        source=CREATE_WEBHOOK,
+        context_value=gql_params.context,
+        root_value=None,
+        variable_values={"name": "builtin-tag-update-1", "node_kind": "BuiltinTag", "event_type": "all"},
+    )
+    assert not result.errors
+    assert result.data
+    webhook_id = result.data["CoreStandardWebhookCreate"]["object"]["id"]
+
+    result = await graphql(
+        schema=gql_params.schema,
+        source=UPDATE_WEBHOOK,
+        context_value=gql_params.context,
+        root_value=None,
+        variable_values={
+            "id": webhook_id,
+            "node_kind": None,
+        },
+    )
+    assert not result.errors
+    assert result.data
+    updated_webhook = await NodeManager.get_one_by_id_or_default_filter(db=db, id=webhook_id, kind=CoreStandardWebhook)
+    assert not updated_webhook.node_kind.value
```

Comment:
` assert updated_webhook.node_kind.value is None`

 make it more explicit about the intent